skimage/data/_fetchers.py

```diff
@@ -152,21 +147,35 @@ def _skip_pytest_case_requiring_pooch(data_filename):
                     allow_module_level=True)
 
 
-def _fetch(data_filename):
+def _ensure_data_dir(*, target_dir):
+    """Prepare local cache directory if it doesn't exist already."""
+    os.makedirs(target_dir, exist_ok=True)
+    readme_src = osp.join(_DISTRIBUTION_DIR, "data/README.txt")
+    readme_dest = osp.join(target_dir, "README.txt")
+    if not osp.exists(readme_dest):
+        shutil.copy2(readme_src, readme_dest)
+
+
+def _fetch(data_filename, *, copy_legacy_to_cache=False):
     """Fetch a given data file from either the local cache or the repository.
 
     This function provides the path location of the data file given
     its name in the scikit-image repository.
 
     Parameters
     ----------
-    data_filename:
+    data_filename : str
         Name of the file in the scikit-image repository. e.g.
         'restoration/tess/camera_rl.npy'.
+    copy_legacy_to_cache : bool, optional
```

Comment:
I think we should decide whether we want this to be True or False, since the user doesn't get to control it. The behavior required (by non-writable Docker) is for this to be False.